app/controllers/api/v1/accounts/notifications_controller.rb

```diff
@@ -3,9 +3,11 @@ class Api::V1::Accounts::NotificationsController < Api::V1::Accounts::BaseContro
 
   before_action :fetch_notification, only: [:update]
   before_action :set_primary_actor, only: [:read_all]
+  before_action :set_current_page, only: [:index]
 
   def index
     @unread_count = current_user.notifications.where(account_id: current_account.id, read_at: nil).count
+    @count = current_user.notifications.where(account_id: current_account.id).count
     @notifications = current_user.notifications.where(account_id: current_account.id).page params[:page]
```

Comment:
Please create a method and memoize `current_user.notifications.where(account_id: current_account.id)`